pages/availability_results.tsx

```diff
@@ -79,22 +80,12 @@ export default function AvailabilityResults(context: NextPageContext) {
             {data && data.results.length} Vaccination Centers
           </h3>
           <div className="flex mobileCol center">
-            <label className={styles.label}>
-              <input
-                list="districts"
-                onChange={(e) => setSearchBarContent(e.target.value)}
-                type="text"
-                className={styles.searchBar}
-                placeholder="Enter your Pincode or District name"
-                value={searchBarContent}
-              />
-              <datalist id="districts">
-                {Districts.map((district) => {
-                  return <option value={district.district_name} />;
-                })}
-              </datalist>
-            </label>
-
+            <SearchDropdown
+              search={searchBarContent}
+              setSearch={setSearchBarContent}
+              listItems={Districts}
+              keyValue="district_name"
```

Comment:
This is not part of the review, but could you explain what the `keyValue` does here?

Reply:
`listItems` is an array of objects. So when the option is selected, it will be used to determine which key value to be set in the `search` state.